docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
       - '8000:8000'
       - '8020:8020'
     environment:
-      postgres_host: 'postgres:5432'
+      postgres_host: 'postgres'
```

Comment:
What's the purpose of this change?

Reply:
so that it works for mac users since the way it was it doesn't